### ISSUE TITLE
Suggestion: Add some screenshots to the README

### ISSUE THREAD

**Issue by sbrl** [nohkumado/openscad_snapfit#1]
This will help people see whether the snap fit designed you've got here are suitable for their build :-)

**Comment by nohkumado** [comment]
i would be more interested in contributions to make this better and more versatile ;)
but here you are!

**Comment by sbrl** [comment]
Indeed, adding new features and fixing bugs is also important, but without good documentation such features are almost useless to someone who comes along and finds your project from outside without any prior knowledge thereof.
